test/unit/local/steps/initial_diff.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-env mocha */
+/* @flow */
+
+const should = require('should')
+const Builders = require('../../../support/builders')
+const configHelpers = require('../../../support/helpers/config')
+const pouchHelpers = require('../../../support/helpers/pouch')
+
+const Buffer = require('../../../../core/local/steps/buffer')
+const initialDiff = require('../../../../core/local/steps/initial_diff')
+
+describe('local/steps/initialDiff', () => {
+  let builders
+  let buffer
+
+  before('instanciate config', configHelpers.createConfig)
+  beforeEach('instanciate pouch', pouchHelpers.createDatabase)
+  beforeEach('populate pouch with documents', function () {
+    builders = new Builders({pouch: this.pouch})
+    buffer = new Buffer()
+  })
+  afterEach('clean pouch', pouchHelpers.cleanDatabase)
+  after('clean config directory', configHelpers.cleanConfig)
+
+  it('detects documents moved while client was stopped', async function () {
+    builders.metadir().path('foo').ino(1).create()
+    builders.metafile().path('baz').ino(2).create()
```

Comment:
blocker: Shouldn't we `await` all those *async* `BaseMetadataBuilder#create()` calls everywhere?

Reply:
We probably do indeed. I forgot to add `await` since the event builders don't need it.
I'll fix that.